open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/DefaultSubjectAreaNodeClients.java

```diff
@@ -109,6 +98,6 @@ public SubjectAreaNode(SubjectAreaRestClient subjectAreaRestClient) {
         if (cache.containsKey(clazz)) {
             return (SubjectAreaNodeClient<T>) cache.get(clazz);
         }
-        return null;
+        throw new IllegalArgumentException("Not found client for " + clazz.getName());
```

Comment:
please use the standard way of doing messages for subject area -add a new entry into SubjectAreaErrorCode and refer to that